Source/WTF/wtf/PlatformHave.h

```diff
@@ -1794,5 +1794,5 @@
 
 // FIXME: Other platforms should finish migration to *.serialization.in.
 #if PLATFORM(COCOA)
-#define ONLY_MODERN_SERIALIZATION 1
+#define HAVE_ONLY_MODERN_SERIALIZATION 1
```

Comment:
What's the state of GTK/WPE with this turned on by default? I'm able to compile 🪟 s with this `ON`.

Reply:
Not ready yet, but I'm working on it.

Reply:
Now it's ready, so I submitted https://github.com/WebKit/WebKit/pull/27734 . Not sure if this is used elsewhere so I marked it as a draft for now.